examples/mvdr_beamformer.h

```diff
@@ -116,14 +116,14 @@ class MVDRBeamformer {
     (covMatView = (covMatView * (1.0f / static_cast<float>(snap_len_))) +
                    eye<complex>() * load_coeff_)
         .run(stream);
-    inv(invCovMatView, covMatView, stream);
+    inv_impl(invCovMatView, covMatView, stream);
```

Comment:
Do we need to rename with _impl?  Or is an overload preferable?

Reply:
Mentioned on slack, but I prefer _impl because it's easier for docs and explicitly separates operator from transform